### PR TITLE
test: add test for daemon rpcserver

### PR DESCRIPTION
**Pull request by power-more**
Signed-off-by: zhaoshang <zhaoshangsjtu@linux.alibaba.com>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: client/daemon/peer/mocks/peertask_manager_mock.go [hunk 1]
```diff
@@ -0,0 +1,446 @@
+// Code generated by MockGen. DO NOT EDIT.
```
Comment (gaius-qi) [inline client/daemon/peer/mocks/peertask_manager_mock.go:1]: User `client/daemon/peer/peertask_manager_mock.go`.

**Review by gaius-qi — CHANGES_REQUESTED** [review]
Fix lint.

File: client/daemon/peer/mocks/peertask_manager_mock.go (hunk 1 above)
Comment (power-more) [reply client/daemon/peer/mocks/peertask_manager_mock.go]: fixed

File: client/daemon/peer/piece_manager_mock.go [hunk 2]
```diff
@@ -0,0 +1,97 @@
+// Code generated by MockGen. DO NOT EDIT.
```
Comment (gaius-qi) [inline client/daemon/peer/piece_manager_mock.go:1]: Add `//go:generate mockgen -destination piece_manager_mock.go -source piece_manager.go -package peer` to piece_manager.go.

**Review by gaius-qi — CHANGES_REQUESTED** [review]

File: client/daemon/peer/piece_manager_mock.go (hunk 2 above)
Comment (power-more) [reply client/daemon/peer/piece_manager_mock.go:1]: fixed

**Review by gaius-qi — APPROVED** [review]
LGTM

**Pull request merged**